src/sentry/runner/commands/upgrade.py

```diff
@@ -23,8 +24,11 @@ def _check_history():
 
     # If we haven't run all the migration up to the latest squash abort.
     # As we squash more history this should be updated.
-    cursor.execute("SELECT 1 FROM django_migrations WHERE name = '0200_release_indices'")
-    if not cursor.fetchone()[0]:
+    cursor.execute(
+        "SELECT 1 FROM django_migrations WHERE name in ('0200_release_indices', '0001_squashed_0200_release_indices')"
```

Comment:
cc @markstory, we might want to put this in a constant, but just unbreaking dev for now